mmv1/products/bigquerybiglake/Database.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: "Database"
+description: |
+  Databases are containers of tables.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Manage open source metadata with BigLake Metastore": "https://cloud.google.com/bigquery/docs/manage-open-source-metadata#create_databases"
+  api: "https://cloud.google.com/bigquery/docs/reference/biglake/rest/v1/projects.locations.catalogs.databases"
+base_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases"
+self_link: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases/{{name}}"
+create_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases?databaseId={{name}}"
+update_mask: true
+update_verb: :PATCH
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "bigquery_biglake_database"
+    primary_resource_id: "database"
+    vars:
+      name: "my_database"
+      catalog_id: "my_catalog"
+      bucket_id: "my_bucket"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "location"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The geographic location where the Database should reside.
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of the database.
+  - !ruby/object:Api::Type::String
+    name: "catalog_id"
```

Comment:
I'd guess this is required & immutable also?

Reply:
done.